thorlcr/graph/thgraph.cpp

```diff
@@ -1174,6 +1174,7 @@ void CGraphBase::init()
 {
     bool log = queryJob().queryForceLogging(queryGraphId(), (NULL == queryOwner()) || isGlobal());
     setLogging(log);
+    setJobAborted(0);
```

Comment:
@jakesmith  Is this the correct place to clear it, or should it be done once abort processing has finished?

Reply:
yes, I think what we want is to turn on this behaviour (prevent cores), once an aborting condition has started, ideally after we know the original error has been reported.
Then turn it off again only when we know it's been handled.

GraphBase::init() is per subgraph, so will be called [too] often during a single [outer] graph.

It's hard to know when the aborting processing has completely finished (globally).
But I think the setJobAborted(0) statement should be called, when the job is completely finished, which is after is at end of CJobBase::endJob()

Reply:
ok, I will move this to endJob().